gtsam_unstable/discrete/tests/testScheduler.cpp

```diff
@@ -75,7 +75,7 @@ DiscreteFactorGraph createExpected() {
   // Mutual exclusion for students
   expected.addAllDiff(A, J);
 
-  return expected;
+  return std::move(expected);
```

Comment:
Ahh! We would have all these for free in C++17 😉
https://en.cppreference.com/w/cpp/language/copy_elision